lib/utils.js

```diff
@@ -223,11 +226,4 @@ async function execSQLiteQuery (db, query, ...queryParams) {
   return await exec('sqlite3', ['-line', db, formattedQuery.join('')]);
 }
 
-async function bindTouchIDKey (bind = true) {
-  // JSHint ignore is to ignore the bad/unnecessary escaping rule
-  ['Toggle Enrolled State', 'Touch ID Enrolled'].forEach(async (key) => 
-    await exec('defaults', ['write', '"Apple Global Domain"', 'NSUserKeyEquivalents', '-dict-add', `"${key}"`, bind ? '"@~\\$^t"': 'nil'])
```

Comment:
I'm guessing the reason teen_process's exec wasn't working is that you have too many quotes in here. teen_process will handle quoting. what if this line were just:

```js
await exec('defaults', ['write', 'Apple Global Domain', 'NSUserKeyEquivalents', '-dict-add', `${key}`, bind ? '@~$^t': 'nil']));
```

(teen_process uses a shlex library which should take care of shell escaping and quoting)